lightning-background-processor/Cargo.toml

```diff
@@ -15,10 +15,10 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
 bitcoin = "0.28.1"
-lightning = { version = "0.0.109", path = "../lightning", features = ["std"] }
-lightning-rapid-gossip-sync = { version = "0.0.109", path = "../lightning-rapid-gossip-sync" }
+lightning = { version = "0.0.110", path = "../lightning", features = ["std"] }
+lightning-rapid-gossip-sync = { version = "0.0.110", path = "../lightning-rapid-gossip-sync" }
 
 [dev-dependencies]
-lightning = { version = "0.0.109", path = "../lightning", features = ["_test_utils"] }
-lightning-invoice = { version = "0.17.0", path = "../lightning-invoice" }
-lightning-persister = { version = "0.0.109", path = "../lightning-persister" }
+lightning = { version = "0.0.110", path = "../lightning", features = ["_test_utils"] }
+lightning-invoice = { version = "0.18.0", path = "../lightning-invoice" }
```

Comment:
Was there a reason to increment the lightning-invoice version? Or do we always increment it?